coalib/results/Result.py

```diff
@@ -264,6 +264,55 @@ def location_repr(self):
         return ', '.join(repr(relpath(range_path))
                          for range_path in sorted(range_paths))
 
+    @enforce_signature
+    def get_affected_source_lines(self,
```

Comment:
actually I think this function should reside in the SourceRange class instead! Then you also don't need to pass filename, start and end line and you'll just use `range.affected_source`, maybe even make it a property.

The way of doctesting you do here is great :)